scripts/apps/archive/directives/MediaMetadata.ts

```diff
@@ -64,6 +63,12 @@ export function MediaMetadata(userList, archiveService, metadata, $timeout) {
 
                 return term.name;
             };
+
+            scope.$watch('label', (newValue) => {
+                if (newValue) {
+                    scope.loading = false;
```

Comment:
Shouldn't this be set at the same place where `scope.label = _getLabelForFieldId;`? `$watch` wouldn't be needed then which would make the code simpler.